go/client/cmd_simplefs_uploads.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	"sort"
+	"time"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSUploads is the 'fs ps' command.
+type CmdSimpleFSUploads struct {
+	libkb.Contextified
+	filter keybase1.ListFilter
+}
+
+// NewCmdSimpleFSUploads creates a new cli.Command.
+func NewCmdSimpleFSUploads(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "uploads",
```

Comment:
feels a little bit odd to me as a name. Maybe `show-uploads` or something, unless you plan more subcommands? Or leave it as is if you disagree, I don't feel strongly.

Reply:
Yeah I see your point, but I think it's consistent with other things like `quota` and `history` -- we don't currently have any fs-level `show-` commands.  Also I imagine we eventually might need a way to cancel uploads, so maybe this will turn into `uploads show` and `uploads cancel`?  Not sure.  But I think I'll keep it as is for now and see how things evolve.